scanner/fetcher.go

```diff
@@ -97,9 +99,10 @@ func (f *Fetcher) Prepare(ctx context.Context) (*ct.SignedTreeHead, error) {
 	if err != nil {
 		return nil, fmt.Errorf("GetSTH() failed: %v", err)
 	}
-	f.Log(fmt.Sprintf("Got STH with %d certs", sth.TreeSize))
+	f.Logger.Printf("Got STH with %d certs", sth.TreeSize)
 	if f.opts.EndIndex == 0 || f.opts.EndIndex > int64(sth.TreeSize) {
 		f.opts.EndIndex = int64(sth.TreeSize)
+		f.Logger.Printf("Reset EndIndex to %d", f.opts.EndIndex)
```

Comment:
Maybe move up a line and change to `"Reset EndIndex from %d to %d"` ?

Reply:
Done.